guides/common/modules/con_package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,25 @@
+[id="package-mode-and-image-mode-hosts_{context}"]
+= Package mode and image mode hosts
+
+You can operate hosts in two modes: package mode and image mode.
+
+{Project} identifies an image mode host as any system that reports a container image from its bootc status through `subscription-manager`. 
+It classifies all other hosts as package mode hosts.
+
+Image mode simplifies enterprise operations by enabling development, operations, and solution providers to use the same container-native tools and techniques to manage both applications and the underlying OS.
+
+The distinction between package mode and image mode provides greater flexibility in managing hosts and streamlines operations for image-based deployments.
+
+{Project} enables registration, monitoring, and update support for image mode hosts. 
+Register image mode hosts like any other host, view their `bootc` statuses in a centralized location, and trigger updates remotely using remote execution. 
+For more information on image mode concepts, refer to the https://www.redhat.com/en/blog/image-mode-red-hat-enterprise-linux-quick-start-guide[RHEL Image Mode quick start guide].
+
+.Image mode hosts
+Image mode hosts include additional fields that reflect their boot status.
+These fields track the container images used in various scenarios:
+
+* *Booted Image & Digest*: Indicates the container image the host is currently using.
+* *Staged Image & Digest*: Indicates the container image the host uses after the next reboot.
+* *Available Image & Digest*: Represents the container image cached by running `bootc upgrade --check`.
+* *Rollback Image & Digest*: Represents the container image the host reverts to after the next reboot if `Bootc rollback` is applied.
+
```

Comment:
Aren't these things described in the web UI?

Reply:
Yes.